File: Lista pt3/Exercicio08Arrays.java
import java.util.LinkedHashSet;
import java.util.Set;

public class Exercicio08Arrays {
    public static void main(String[] args) {

        int[] numeros = {
                10, 20, 30, 20, 40,
                50, 10, 60, 70, 30
        };

        Set<Integer> semRepetidos =
                new LinkedHashSet<>();

        for (int numero : numeros) {
            semRepetidos.add(numero);
        }

        System.out.println(
                "Valores sem repetição:"
        );

        for (int numero : semRepetidos) {
            System.out.println(numero);
        }
    }
}

File: Lista pt3/Exercicio08Arrays.js
let numeros = [
    10,
    20,
    30,
    20,
    40,
    50,
    10,
    60,
    70,
    30
];

let semRepetidos = [];

for (let i = 0; i < numeros.length; i++) {

    if (!semRepetidos.includes(numeros[i])) {
        semRepetidos.push(numeros[i]);
    }
}

console.log("Array original:");

console.log(numeros);

console.log("Array sem números repetidos:");

console.log(semRepetidos);